components/layout/Navbar.tsx

```diff
@@ -76,9 +80,9 @@ export function Navbar({}) {
               </li>
               {data.map((item) => {
                 const navLinkClasses =
```

Comment:
Do we need a variable for this?

Reply:
Was not apart of my implementation (i.e was there prior - i can move it out perhaps? I think its fine where it is

Reply:
yea seems weird - lets use it in classnames, move it to a component or use 1 variable at the top. Cause they seem to be the same

Reply:
✅